### PR TITLE
docs(bugbounty): add yarn as eligible for bounty

### PR DESCRIPTION
**Pull request by lirantal**
I reached out to @arcanis to ask for his approval to add yarn to our eligible bounties and he gave his blessings ❤️

### CODE REVIEW

File: processes/bug_bounty_criteria.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ resolve security reports.
 * [fastify](https://github.com/fastify/fastify) (confirmed approval from Matteo Collina)
 * [pino](https://github.com/pinojs/pino) (confirmed approval from Matteo Collina)
 * [MQTT.js](https://github.com/mqttjs/MQTT.js) (confirmed approval from Matteo Collina)
+* [yarn](https://github.com/yarnpkg/yarn) (confirmed approval from Maël)
```
Comment (cjihrig) [inline processes/bug_bounty_criteria.md]: Can you include the last name of the approver.

Comment (lirantal) [reply processes/bug_bounty_criteria.md]: Yes, I've updated it.

**Comment by reedloden** [comment]
@lirantal Might be good to confirm that yarn wouldn't be covered under Facebook's Whitehat Program. https://www.facebook.com/whitehat states that its scope includes:
> Open source projects by Facebook (e.g. osquery)

Yarn was [created by Facebook](https://engineering.fb.com/web/yarn-a-new-package-manager-for-javascript/).

**Comment by lirantal** [comment]
@reedloden good point but I don't think Yarn is governed by Facebook.
@arcanis can you confirm?

**Comment by arcanis** [comment]
Yarn was created by *people from* Facebook, but also other places. More about this [here](https://next.yarnpkg.com/advanced/qa#is-yarn-operated-by-facebook).

Our size, roadmap, and history currently make us closer from a BDFL model, although I hope to formalise a bit more our governance once we're past the v2 release.

**Pull request merged**